posthog/clickhouse/materialized_columns.py

```diff
@@ -29,3 +29,23 @@ def get_materialized_column_for_property(
         table: TablesWithMaterializedColumns, table_column: TableColumn, property_name: PropertyName
     ) -> MaterializedColumn | None:
         return None
+
+
+ADD_COLUMN_SHARDED_EVENTS_SQL = """
```

Comment:
I know you just moved this and didn't write this, but the name is now pretty confusing. It made some sense in the migration file, but here it probably needs renaming